lib/plugins/aws/package/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -159,7 +159,7 @@ module.exports = {
         );
 
         const integrationResponse = {
-          StatusCode: parseInt(statusCode, 10),
+          StatusCode: parseInt(statusCode, 10).toString(),
```

Comment:
Let's just put `statusCode` (note that in original code string is converted to number, instead of reconverting obtained number to string, let's just not convert to number)

Reply:
Right! Missed that 👍